### PR TITLE
Added command status.

### PR DESCRIPTION
**Pull request by SkyToGround**
As the title says: two classes for keeping track of (write) jobs.

### CODE REVIEW

File: tests/test_command_status.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+def test_eq_failure_1():
+    under_test1 = CommandStatus("job_id1", "command_id1")
+    under_test2 = CommandStatus("job_id2", "command_id1")
+    assert under_test1 != under_test2
+
+
+def test_eq_failure_2():
+    under_test1 = CommandStatus("job_id1", "command_id1")
+    under_test2 = CommandStatus("job_id1", "command_id2")
+    assert under_test1 != under_test2
```
Comment (umithardal) [inline tests/test_command_status.py:34]: These two tests, failure 1&2, can be combined by using pytest parametrisation. See eg [here](https://docs.pytest.org/en/stable/parametrize.html).

File: file_writer_control/CommandStatus.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    The status of a command.
+    """
+    def __init__(self, job_id: str, command_id: str):
+        self._job_id = job_id
+        self._command_id = command_id
+        self._last_update = datetime.now()
+        self._state = CommandState.NO_COMMAND
+        self._message = ""
+
+    def __eq__(self, other):
```
Comment (umithardal) [inline file_writer_control/CommandStatus.py:28]: May be we can give more meaningful names to the args.

**Comment by umithardal** [comment]
In general the tests are built for known cases. We are not testing against possible edges. May be there are none for these operations but it is a good thing to create some tests for possible crazy scenarios.

File: file_writer_control/CommandStatus.py (hunk 2 above)
Comment (umithardal) [reply file_writer_control/CommandStatus.py:28]: Also may be provide the type for the arg `other`?

Comment (SkyToGround) [reply file_writer_control/CommandStatus.py:28]: I missed the type here, it has now been added. Feel free to suggest a better name for the argument.

File: tests/test_command_status.py (hunk 1 above)
Comment (SkyToGround) [reply tests/test_command_status.py:34]: A fair suggestion. However, when implementing this I noticed that parametrisation in this case resulted in more code (about 15% more). Hence I suggest that parametrisation not be done in this particular case. Still, a method to keep in mind for the future.

Comment (umithardal) [reply tests/test_command_status.py:35]: I would prefer the longer code to a repeating one. However, if you feel that this is more optimal, sure.

File: file_writer_control/CommandStatus.py (hunk 2 above)
Comment (umithardal) [reply file_writer_control/CommandStatus.py]: I would have used `other_status`, as it is basically a status and we are checking if it is an instance of `CommandStatus`. But `other` is fine, if you prefer it.

Comment (SkyToGround) [reply file_writer_control/CommandStatus.py]: I think that `other_status` is redundant, especially after having added the type to the function declaration. I have changed it though.

**Review by umithardal — APPROVED** [review]

**Pull request merged**